web-portal/components/DateTimePicker.vue

```diff
@@ -9,13 +9,7 @@
     </div>
     <div class="time-date-control-wrapper">
       <div>
-        <v-date-picker
-          v-model="picker"
-          color="gray lighten-1"
-          :allowed-dates="AllowedDates"
-          :no-title="true"
-          :dark="true"
-        />
+        <date-picker @change="dateChanged" />
```

Comment:
How does this get populated with the selection when editing a datetime?

Reply:
not sure I follow, I guess the answer is it doesn't. It goes the other direction. You pick a date in this component which fires the dateChanged and that sets a new time. Is there any reason this needs to work the other way?

Reply:
fixed 8b00e02e2bba94b34d4235264f7215cdb46a9131